README.md

```diff
@@ -7,6 +7,7 @@ We utilize puppeteer to get chrome performance traces.
 
 To run all you have to do is start the server and run the test suite on it.
 ```
+npm run test-all

```

Comment:
Whether to use "test-all" or "test" is a choice for each, so I want each to be written in a separate block.